larq/layers.py

```diff
@@ -201,6 +208,11 @@ def __init__(
         bias_constraint=None,
         **kwargs,
     ):
+        if groups != 1:
+            if version.parse(tf.__version__) >= version.parse("2.3"):
+                kwargs = {**kwargs, "groups": groups}
+            else:
+                raise ValueError("`groups` != 1 requires TensorFlow version 2.3 or newer.")
```

Comment:
I don't really like the added boilerplate here, but I think it is nicer to have the groups parameter available in the signature and not hidden in the `kwargs`.